web/components/FcDrawerViewCollisionReports.vue

```diff
@@ -124,6 +178,12 @@ export default {
     };
   },
   computed: {
+    activeLocations() {
```

Comment:
rename to `locationsActive` - it's confusing to have `activeLocations` _and_ `locationActive`, and the pluralization could easily be missed by future readers of this code